jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -1011,8 +1012,9 @@ private static void putSanitisedValue(String s, ByteBuffer buffer)
         {
             char c = s.charAt(i);
 
-            if (c > 0xff || c == '\r' || c == '\n')
-                buffer.put((byte)' ');
+            // Do not allow 8-bit, or CTRL (except HTAB)
+            if (c > 0xff || (c >= 0x00 && c <= 0x1F && c != '\t'))
+                buffer.put((byte)' '); // replacement char has to satisfy RFC9110 Field Value ABNF.
```

Comment:
Space seams like a poor choice, but let's not change for now.
Again feels like a compliance mode that would allow a hard fail here would be good.